Autopilot/Inc/SensorFusion.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong, Dhruv Rawat, Anthony Bertnyk
+ */
+
+#include "AttitudeDatatypes.hpp"
+#include "fetchSensorMeasurementsMode.hpp"
+#include "arm_math.h"
+
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+typedef struct {
+    float IMUroll, IMUpitch, IMUyaw; //in rad (for now)
+    float IMUrollrate, IMUpitchrate, IMUyawrate; //in rad/s (for now)
+    float Airspeed; //in m/s (for now)
+} SFOutput_t ;
+
+typedef struct {
+    float altitude;
+    long double latitude; 
+    long double longitude;
+} SFPositionOutput_t;
+
+typedef struct {
+    arm_matrix_instance_f32 prevX;
+    arm_matrix_instance_f32 prevP;
+} SFIterationData_t;
+
+// -1 = FAILED
+// 0 = SUCCESS
+// 1 = Old Data
+struct SFError_t{
+    int errorCode;
+};
+
+//The SF_Get functions ensure SensorFusion does not have access to the sensor drivers
+/**
+ * Get attitude data by fusing data from IMU and airspeed sensors.
+ * @param[out]  Output  Reference to an output struct for attitude data.
+ * @param[in]   imudata
+ * @param[in]   airspeeddata
+ */ 
+SFError_t SF_GetResult(SFOutput_t *Output, IMU_Data_t *imudata, Airspeed_Data_t *airspeeddata);
```

Comment:
Perhaps use a more descriptive name for this function? The other function is called SF_GetPosition.

Perhaps, since SF_GetResult is used by Attitude Manager and SF_GetPosition is used by Path Manager, we could name them SF_AttitudeManager_Result and SF_PathManager_Result? It'll also make it clear to future people which parts of our code call each function.